action.php

```diff
@@ -24,6 +21,8 @@ public function register(Doku_Event_Handler $controller)
     /**
      * Manage page meta data
      *
+     * FIXME do we track ALL pages? or do we check for assignments?
+     *
```

Comment:
Yeah, I wouldn't bother with trying to only track the assignment ones. We may want to add some indexes on the tables to speed thing up a little, though.

Reply:
ok